app/controllers/queues_controller.rb

```diff
@@ -0,0 +1,56 @@
+class QueuesController < ApplicationController
+  def dedupe
+    @original = RescueRequest.find_by(dupe_of: nil)
+    if @original.nil?
+      redirect_to :disasters
```

Comment:
I'd suggest setting a flash info message here to tell the user *why* they've just been redirected back to disasters - something like `flash[:info] = "There are no dedupe reviews available."`.